website/src/panel-layout.tsx

```diff
@@ -143,6 +183,20 @@ export const PanelLayout = (p: {
   return (
     <div className={css.wordPanelContent}>
       <>{panel}</>
+      {commentsPanel === false && (
+        <Button type="button" onClick={handleComment}>
```

Comment:
```suggestion
        <Button type="button" onClick={() => setCommentsPanel(!commentsPanel}>
```

Reply:
This looks like it should only ever open the comments panel, right? Or this button shared between the open/closed states somehow?